notebooks/ch18/hyp_studentized.md

```diff
@@ -0,0 +1,415 @@
+
+
+```python
+# HIDDEN
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+%matplotlib inline
+import ipywidgets as widgets
+from ipywidgets import interact, interactive, fixed, interact_manual
+import nbinteract as nbi
+
+sns.set()
+sns.set_context('talk')
+np.set_printoptions(threshold=20, precision=2, suppress=True)
+pd.options.display.max_rows = 7
+pd.options.display.max_columns = 8
+pd.set_option('precision', 2)
+# This option stops scientific notation for pandas
+# pd.set_option('display.float_format', '{:.2f}'.format)
+```
+
+
+```python
+# HIDDEN
+def df_interact(df, nrows=7, ncols=7):
+    '''
+    Outputs sliders that show rows and columns of df
+    '''
+    def peek(row=0, col=0):
+        return df.iloc[row:row + nrows, col:col + ncols]
+    if len(df.columns) <= ncols:
+        interact(peek, row=(0, len(df) - nrows, nrows), col=fixed(0))
+    else:
+        interact(peek,
+                 row=(0, len(df) - nrows, nrows),
+                 col=(0, len(df.columns) - ncols))
+    print('({} rows, {} columns) total'.format(df.shape[0], df.shape[1]))
+```
+
+
+```python
+# Generate a bunch of random points. To change the dataset, just
+# change this line and rerun notebook
+times = (np.random.exponential(scale=300, size=13825) +
+         np.abs(np.random.normal(loc=50, scale=40, size=13825)))
+```
+
+## The Studentized Bootstrap
+
+
+
+The **bootstrap** is a process we learned about in Data 8 that we can use for estimating a population statistic using only one sample. The general procedure for bootstrapping is as follows:
+- Draw a sizable sample from the population.
+- Using this sample, we *resample with replacement* to form new samples of the same size.
+- We do this many times, taking the desired statistic of each resample (e.g. we take the median of each resample).
+
+Here, we end up with many test statistics from individual resamples, from which we can form a distribution. In Data 8, we were taught to form a confidence interval by taking the 2.5th percentile and the 97.5th percentile. This method of bootstrapping to create a confidence interval is called the **percentile bootstrap.** Generally speaking, this confidence interval will have a 95% chance of containing the actual population statistic. It is important to note, however, that this will NOT always be the case. 
```

Comment:
We should be explicit and say "form an approximate 95% confidence interval by taking the 2.5th percentile and the 97.5th percentile of the bootstrapped test statistics."

The second-to-last sentence is true but is misleadingly worded. Given a confidence interval, the population parameter is either in the interval or it is not. However, there is a 95% probability that if we draw a new sample from the population and construct a CI that the interval will contain the parameter. I suggest the following rewording of the last two sentences:

> 95% confidence implies that if we take a new sample from the population and construct a confidence interval, the confidence interval will contain the population parameter with probability 0.95. However, it is important to note that confidence intervals created from real data can only approximate 95% confidence. The percentile bootstrap in particular has lower confidence than desired at small sample sizes.